src/nsis.mk

```diff
@@ -4,8 +4,8 @@ PKG             := nsis
 $(PKG)_WEBSITE  := https://nsis.sourceforge.io/
 $(PKG)_DESCR    := NSIS
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 3.05
-$(PKG)_CHECKSUM := b6e1b309ab907086c6797618ab2879cb95387ec144dab36656b0b5fb77e97ce9
+$(PKG)_VERSION  := 3.04
+$(PKG)_CHECKSUM := 609536046c50f35cfd909dd7df2ab38f2e835d0da3c1048aa0d48c59c5a4f4f5
```

Comment:
Move this and the scons to a new PR and base your changes after the parallel build fix.